src/quo/components/counter/collectible_counter/style.cljs

```diff
@@ -0,0 +1,72 @@
+(ns quo.components.counter.collectible-counter.style
+  (:require
+    [quo.foundations.colors :as colors]))
+
+(defn- get-background-color
+  [{:keys [status theme]}]
+  (case status
+    :default (colors/theme-colors (colors/resolve-color colors/white-opa-70 theme)
+                                  (colors/resolve-color colors/neutral-95-opa-70 theme)
+                                  theme)
+    :error   (colors/theme-colors (colors/resolve-color colors/danger-50-opa-10 theme)
+                                  (colors/resolve-color colors/danger-60-opa-10 theme)
+                                  theme)
+    (colors/theme-colors (colors/resolve-color colors/white-opa-70 theme)
+                         (colors/resolve-color colors/neutral-95-opa-70 theme)
+                         theme)))
+
+(defn- get-container-border-styles
+  [{:keys [status theme]}]
+  (when (= status :error)
+    {:border-color (colors/theme-colors (colors/resolve-color colors/danger-50-opa-20 theme)
```

Comment:
this should be:
```
(colors/resolve-color :danger-50 theme 20)
```

 _Shouldn't colors/danger-60-opa-20 be here?_
 Nope, if you look how the designers are using the colors, you'll see that danger, success (and basically any 50/60 color values) are using the -50 when there is opacity added. 
 The resolve color method handles this internally so that's why it's better to use that method as we abstract away this logic :)